package.json

```diff
@@ -104,9 +104,6 @@
   "devDependencies": {
     "@babel/core": "^7.1.0",
     "@babel/preset-env": "^7.2.0",
-    "@storybook/addon-info": "4.0.11",
-    "@storybook/addons": "^4.0.11",
-    "@storybook/react": "^4.0.11",
```

Comment:
Are these storybook dependencies just for development and we weren't using them? Can you include details in PR overview

Reply:
Storybook is not used in the public repo; these dependencies are not used.